cuML/examples/kmeans/README.md

```diff
@@ -0,0 +1,80 @@
+# kmeans
+This subfolder contains an example on how to use cuML kmeans from C++ application
+There are two `CMakeLists.txt` in this folder:
+1. `CMakeLists.txt` (default) which is included when building cuML
+2. `CMakeLists_standalone.txt` as an example for a stand alone project linking to `libcuml.so`
+
+## Build
+`kmeans_example` is build as part of cuML. If it should be build as a standalone executable use `CMakeLists_standalone.txt` and configure with:
+```bash
+$ cmake .. -DCUML_LIBRARY_DIR=/path/to/directory/with/libcuml.so -DCUML_INCLUDE_DIR=/path/to/directory/with/kmeans/kmeans_c.h
```

Comment:
Small question, do we want to put the output of the cmake in the instructions? It will vary based on the particular environment of a user, so I'd recommend dropping it.

Reply:
Sure I will change that.